### PR TITLE
chore: Fix unit tests by increasing coverage of tooltip to 100%

### LINKED ISSUE #7585: Unit tests are failing due to <100% coverage of the new @k8slens/tooltip

### PR DESCRIPTION
**Pull request by Nokel81**
fixes https://github.com/lensapp/lens/issues/7585

### CODE REVIEW

File: packages/ui-components/tooltip/src/__snapshots__/tooltip.test.tsx.snap [hunk 1]
```diff
@@ -49,3 +51,195 @@ exports[`<Tooltip /> renders to DOM when hovering over target 1`] = `
...
+  </div>
+  <div
+    class="Tooltip right visible"
+    data-testid="tooltip"
+    role="tooltip"
+    style="left: 10px; top: 0px;"
+  >
+    I am a tooltip
+  </div>
+</body>
```
Comment (jim-docker) [inline packages/ui-components/tooltip/src/__snapshots__/tooltip.test.tsx.snap:74]: What in here demonstrates `usePortal` has been applied?

**Review by gabriel-mirantis — APPROVED** [review]

File: packages/ui-components/tooltip/src/__snapshots__/withTooltip.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+`;
+
+exports[`withTooltip tests renders a tooltip when specified without id 1`] = `
+<body>
+  <div>
+    <div>
+      foobar
+    </div>
+  </div>
+</body>
```
Comment (jim-docker) [inline packages/ui-components/tooltip/src/__snapshots__/withTooltip.test.tsx.snap:40]: tooltip supposed to be in these snapshots?

File: packages/ui-components/tooltip/src/__snapshots__/tooltip.test.tsx.snap (hunk 1 above)
Comment (Nokel81) [reply packages/ui-components/tooltip/src/__snapshots__/tooltip.test.tsx.snap:74]: The tooltip is at the root of the DOM, not next to the target like the JSX might imply.

File: packages/ui-components/tooltip/src/__snapshots__/withTooltip.test.tsx.snap (hunk 2 above)
Comment (Nokel81) [reply packages/ui-components/tooltip/src/__snapshots__/withTooltip.test.tsx.snap:40]: Ah good point

**Pull request merged**